README.md

```diff
@@ -0,0 +1,25 @@
+# Colosseum 
+
+## Setup Husky
```

Comment:
We will probably need to move this into DOCUMENTATION.md

Reply:
renamed to DOCUMENTATION.md